ztp/kube-compare-reference/optional/storage/StoragePVC.yaml

```diff
@@ -0,0 +1,7 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pvc-storage
+  namespace: default
+spec:
+{{ .spec | toYaml | indent 2 }}
```

Comment:
This CR is not included in metadata.yaml because we don't make any assertions on it's content. I'd recommend adding it to the metadata.yaml but leave it commented out and add a comment saying that we don't assert any of it's content so validation is not needed. That way someone coming later knows we didn't just miss it.